efm32hg-blinky/Makefile

```diff
@@ -132,7 +132,7 @@ deps:
 # REAL TARGETS // COMMON
 #
 GECKO_A_SRC = Gecko_SDK/Device/SiliconLabs/EFM32HG/Source/GCC/startup_efm32hg.S
-GECKO_C_SRC = $(shell find Gecko_SDK/emlib/src -name \*.c) \
+GECKO_C_SRC = $(shell [ -d Gecko_SDK ] && find Gecko_SDK/emlib/src -name \*.c) \
```

Comment:
Hey! Sorry, I didn't see your comment from two days ago. I re-added the comments with the toolchain location, I removed them because it's in the README but if you feel they should be in there as well...
And yes, this is needed to allow make deps to work. Well, it will "work" if this isn't here, but it will error, so...